src/widget/wstemlabel.cpp

```diff
@@ -0,0 +1,51 @@
+#include "wstemlabel.h"
+
+#include "moc_wstemlabel.cpp"
+
+WStemLabel::WStemLabel(QWidget* pParent)
+        : WLabel(pParent),
+          m_stemInfo(QString(), QColor()),
+          m_stemNo(0) {
+}
+
+void WStemLabel::setup(const QDomNode& node, const SkinContext& context) {
+    m_stemNo = context.selectInt(node, "StemNum");
```

Comment:
assert that num is >= 0 and < N (what's the upper limit?)
otherwise we'd get `ASSERT failure in QList::operator[]: "index out of range"`